RocketChat/Rocket.Chat
# [FIX] Discussion alphabetical  ordering

## Proposed changes (including videos or screenshots)


Added a validation in the prop used for sorting (loweCaseName) checking for a prop that only exists in discussions (prid)



## Issue(s)


## Steps to test or reproduce
Steps to reproduce:

1. Log in to the [rocket.chat](http://rocket.chat/) (version 4.8.0-rc.1)
2. Click on the button Display and select the option Sort by : Name 

Actual result:

-   The discussions were not sorted by name in all types of Group: Unread, Favorites and Types 

![image](https://user-images.githubusercontent.com/20212776/172384754-3d33c76b-926b-4814-bb2f-22c7f0f914ae.png)

Expected result:
The discussions should be sorted by name

## Further comments
